### PR TITLE
Issue5

### PR DESCRIPTION
**Pull request by kdeng121**

### CODE REVIEW

**Comment by kdeng121** [comment]
[Issue 5: Image Tags](https://trello.com/c/fNqlLEMV/62-issue-5-image-tags)

File: test/controllers/images_controller_test.rb [hunk 1]
```diff
@@ -25,28 +25,37 @@ class ApplicationControllerTest < ActionDispatch::IntegrationTest
...
     assert_select 'h1', 'All Images'
     assert_select 'img' do |img|
       assert_equal 'http://test3.com', img[0].attribute('src').value
       assert_equal 'http://test2.com', img[1].attribute('src').value
       assert_equal 'http://test1.com', img[2].attribute('src').value
     end
+    assert_select '.js-tag' do |imgtags|
+      assert_equal 'tagtest31, tagtest32', imgtags[0].children[0].text.strip
+      assert_equal 'tagtest21, tagtest22', imgtags[1].children[0].text.strip
+      assert_equal 'tagtest11, tagtest12', imgtags[2].children[0].text.strip
```
Comment (raj-al-ghul) [inline test/controllers/images_controller_test.rb]: maybe include a test case with no tags?

File: app/views/images/new.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <h1>New Image</h1>
 <%= simple_form_for @image do |f| %>
   <%= f.input :url %>
+  <%= f.input :tag_list %>
```
Comment (raj-al-ghul) [inline app/views/images/new.html.erb]: can we add a test for the create action as well?

File: app/models/image.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
 class Image < ApplicationRecord
   validates :url, presence: true, format: URI.regexp(%w[http https])
+  acts_as_taggable_on :tags
```
Comment (raj-al-ghul) [inline app/models/image.rb]: can we add to the model test?

File: test/controllers/images_controller_test.rb (hunk 1 above)
Comment (kdeng121) [reply test/controllers/images_controller_test.rb]: Changed first image to have no tags

**Comment by raj-al-ghul** [comment]
can we try to add tests to the commits that add the functionality?

File: app/views/images/new.html.erb (hunk 2 above)
Comment (kdeng121) [reply app/views/images/new.html.erb]: Added tag test inside create action test

File: app/models/image.rb (hunk 3 above)
Comment (kdeng121) [reply app/models/image.rb]: Added 'Accepts tag' test to Image model test

File: test/models/image_test.rb [hunk 4]
```diff
@@ -13,8 +13,14 @@ class ImageTest < ActiveSupport::TestCase
     assert_equal ['Url is invalid'], image.errors.full_messages
   end
 
-  test 'Valid when URL is valid' do
+  test 'Valid when URL is valid and tags accepted' do
```
Comment (raj-al-ghul) [inline test/models/image_test.rb]: can we remove this change?

Comment (kdeng121) [reply test/models/image_test.rb]: Removed

**Pull request merged**